### PR TITLE
Feature | Case studies page

### PR DESCRIPTION
**Pull request by monsteronfire**
This PR includes:
- a stand-alone page for case studies
- a new CTA row on the homepage that links to the case studies page
- some CSS clean up of "portfolio" related code
- elements to hold client testimonials/quotes

### CODE REVIEW

**Comment by calumhalcrow** [comment]
Thanks Shaii, I took an initial look and this looks great. I'll take a fully look shortly.

**Comment by monsteronfire** [comment]
No problem, @calumhalcrow. I'm working on the EssayJack content in notepad. Will transfer it to the markdown file once I'm done.

File: _case_studies/essayjack.markdown [hunk 1]
```diff
@@ -13,21 +13,35 @@ project-url: https://www.essayjack.com
...
+
+The next order of business was stabilize the product. This stabilization process included a period of further bug fixing. We got the test suite into a state where it could be relied on. We expanded it’s coverage and hooked it into our CI process. To solve certain performance issues and make deployment more convenient, we migrated the app to AWS OpsWorks.
+
+With a reliable test suite in place, we began refactoring. The first major area we tackled was the template editor. Rebuilding this key feature was a crucial step in moving the product into a phase where it was open for extension. The refactoring put us in a good place for a front-end revamp to be considered.
+
+A design firm called Digital Natives were brought in at this point to design a brand new app for EssayJack. Our goal was to bring this reimagined EssayJack to life. The complex essay editor had a lot of moving parts, so we needed a solution that would give us a way to manage state and provide us with a view layer. We went with React.
```
Comment (calumhalcrow) [inline _case_studies/essayjack.markdown]: Link to Digital Natives homepage? Might help their SEO _ducks_

Comment (monsteronfire) [reply _case_studies/essayjack.markdown]: Updated link

**Comment by calumhalcrow** [comment]
Hi Shaii, since the "do-not-merge" tag is off, here's some feedback. All of it minor polishing stuff:

Probably better to reword the tagline for the EJ case study as seen on the main Case Studies page: "Clunky beta app to new React-based app". 

Shall we remove the Avvo Case Study from the main Case Studies page until we have one?

Looks like the tags are coming through without a space after the comma: "Tags: Ruby,React,ProjectRescue".

There's a "pending" in the heading description:

> Pending. One of the things we pride ourselves in is listening to our clients to truly understand their needs. We prefer to work with a smaller number of clients long term

**Comment by monsteronfire** [comment]
@calumhalcrow All comments have been resolved.

**Pull request merged**